pb-jelly/Cargo.toml

```diff
@@ -16,4 +16,5 @@ categories = ["encoding", "parsing", "web-programming"]
 [dependencies]
 byteorder = "1.4"
 bytes = "1.0"
+compact_str = { version = "0.5", features = ["bytes"] }
```

Comment:
This seems like a good thing to put behind a feature flag - to minimize dependencies on the standard distribution.

Reply:
I like the idea, but then it would require folks to coordinate a feature between `pb-jelly` and their proto files, whereas today they just need to define it in their proto files. What do you think of that extra step?